src/gettingstarted/cli.md

```diff
@@ -75,6 +75,54 @@ Options:
  --shell (-s)          Launch the shell
 ```
 
+### CLI features
+
+Additional settings to control the operation of the `platform` CLI can be managed in the configuration file (See `.platform/local/project.yaml`) or environment variables. See the [`platformsh-cli` `README` for details](https://github.com/platformsh/platformsh-cli/blob/master/README.md#usage). 
+
+#### Auto-selecting your project
+
+When your shells working directory is inside a local checkout of your project repository, the `platform` CLI tool will autodetect your project ID and environment, so you don't need to list them as parameters each time.
+
+Eg, in your home directory you need to provide the project ID as an argument each time:
```

Comment:
In your home directory, for example, you need to provide the project ID as an argument each time: